src/Miunie.Discord/CommandModules/CurrencyCommand.cs

```diff
@@ -19,28 +19,28 @@ public CurrencyCommand(EntityConvertor convertor, CurrencyService currencyServic
         }
 
         [Command("CZK")]
-        public async Task ShowCzkStatus(CommandContext ctx)
+        public async Task ShowCzkStatus()
```

Comment:
All these methods are async but they don't have the Async suffix, for consistency we should do that perhaps?

Reply:
Just checked the ProfileCommands and they do have the suffix, I'd say +1 to this then.